<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>龙少软件大全</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>430</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 242, 249);
border-radius: 5px;
font-family: 'Microsoft YaHei';</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="toFindBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>215</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    font-size: 12px;
	color: rgb(57, 143, 230);
}
QPushButton:hover{
	color: rgb(111, 186, 248);
}</string>
      </property>
      <property name="text">
       <string>找回密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toAlterBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>245</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    font-size: 12px;
	color: rgb(57, 143, 230);
}
QPushButton:hover{
	color: rgb(111, 186, 248);
}</string>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pwdLineEdit">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>215</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 0px;
    border-top-right-radius: 0px;
	border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
      <property name="maxLength">
       <number>18</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toRegisterBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>185</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    font-size: 12px;
	color: rgb(57, 143, 230);
}
QPushButton:hover{
	color: rgb(111, 186, 248);
}</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>285</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="remeberCheckBox">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>255</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8D9195;</string>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="autoLoginCheckBox">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>255</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8D9195;</string>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>185</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	border-bottom-left-radius: 0;
    border-bottom-right-radius: 0;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	border: 1px solid #1583DD;
}</string>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>180</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-bottom-left-radius: 0;
border-bottom-right-radius: 0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="closeBtn">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	image: url(:/Icon/close2.png);
	border-radius: 0;
}
QPushButton:hover{
	image: url(:/Icon/close.png);
	background-color: rgb(255, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="miniBtn">
       <property name="geometry">
        <rect>
         <x>374</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	image: url(:/Icon/mini2.png);
	border-radius: 0;
}
QPushButton:hover{
	image: url(:/Icon/mini.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>251</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 0;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ffffff;&quot;&gt;龙少软件大全&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="headPortraitLabel">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>185</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/Icon/user.png);
background-color: rgb(218, 219, 221);
border-radius:40px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="registerUsernameLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;账号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="registerPwdLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="registerPwd2Label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;确认密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerUsernamelineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPwdlineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPwd2lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerBtn">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerReturnLoginBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="registerMobileLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;手机号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerMobileLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 0px;
    border-top-right-radius: 0px;
	border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLineEdit" name="alterUsernamelineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="alterOldPwdlineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="alterNewPwdlineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="alterOldPwdLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;旧密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alterReturnLoginBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alterBtn">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>220</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="alterUsernameLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;账号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="alterNewPwdLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;新密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="alterNewPwd2Label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;确认密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="alterNewPwd2lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 0px;
    border-top-right-radius: 0px;
	border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="alterMobileLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;手机号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="alterMobileLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLineEdit" name="findUsernameLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findMobileLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findReturnLoginBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="findMobileLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;手机号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="findNewPwd2Label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;确认密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="findUsernameLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;账号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="findNewPwdLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;新密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findNewPwdLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 0px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findNewPwd2LineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>195</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius: 0px;
    border-top-right-radius: 0px;
	border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    font-size: 13px;
    color: black;
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
    border: 1px solid #1583DD;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findBtn">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #00A3FF;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #3CC3F5;
}</string>
      </property>
      <property name="text">
       <string>找回</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>miniBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>lower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
